package-lock.json

```diff
@@ -1274,6 +1274,11 @@
       "integrity": "sha512-nne9/IiQ/hzIhY6pdDnbBtz7DjPTKrY00P/zvPSm5pOFkl6xuGrGnXn/VtTNNfNtAfZ9/1RtehkszU9qcTii0Q==",
       "dev": true
     },
+    "acorn": {
```

Comment:
This file shouldn't need any changes so we should revert this.

Reply:
I am not sure where these changes came from as we did nothing to edit the package-lock.json